app/src/main/java/com/github/wnder/ImageFromGalleryActivity.java

```diff
@@ -21,6 +18,7 @@
     private ImageView imageSelected;
     private Button confirmButton;
     private static final int SELECT_IMAGE = 0;
+    public static final String HAS_SUCCEEDED = "success";
```

Comment:
Soudln't this variable be in the uploadActivity since it is the activity where we need to go through to upload a picture?

Reply:
I followed what is done in the bootcamp to pass extra variables with the intents.